src/ploneintranet/suite/tests/lib/keywords.robot

```diff
@@ -97,7 +110,326 @@ I go to the Service Announcements Workspace
+    Click link  link=Security
+    Wait until page contains  Workspace policy
+
+I can open the workspace member settings tab
+    Click Link  link=Workspace settings and about
+    Click link  link=Members
+    Wait until page contains  Members
+
+I can set the external visibility to Open
+    Comment  AFAICT selenium doesn't yet have support to set the value of a range input field, using JavaScript instead
+    Execute JavaScript  jQuery("[name='external_visibility']")[0].value = 3
+    Submit form  css=#sidebar-settings-security
+    Wait Until Page Contains  Security
+    Click link  link=Security
+    Wait until page contains  The workspace can be explored by outsiders.
+
+I can set the join policy to Admin-Managed
+    Comment  AFAICT selenium doesn't yet have support to set the value of a range input field, using JavaScript instead
+    Execute JavaScript  jQuery("[name='join_policy']")[0].value = 1
+    Submit form  css=#sidebar-settings-security
+    Wait Until Page Contains  Security
+    Click link  link=Security
+    Wait until page contains  Only administrators can add workspace members.
+
+I can set the participant policy to Moderate
+    Comment  AFAICT selenium doesn't yet have support to set the value of a range input field, using JavaScript instead
+    Execute JavaScript  jQuery("[name='participant_policy']")[0].value = 4
+    Submit form  css=#sidebar-settings-security
+    Wait Until Page Contains  Security
+    Click link  link=Security
+    Wait until page contains  Workspace members can do everything
+
+I can see upcoming events
+    Page Should Contain Element  xpath=//a[.='Plone Conf']
+
+Older events are hidden
+    Element should not be visible  jquery=div#older-events a
+
+I can delete an old event
+    Click Element  css=div#older-events h3
+    Mouse Over  xpath=//div[@id='older-events']//li[@class='cal-event']
+    Focus  xpath=//div[@id='older-events']//li[@class='cal-event']
+    Click Element  css=div#older-events button[type='submit']
+    Wait Until Page Contains  Do you really want to delete this item
+    Click Button  Delete
+
+I can go to the sidebar tasks tile
+    Go To  ${PLONE_URL}/workspaces/open-market-committee
+    Click Link  link=Tasks
+    Wait Until Element Is visible  xpath=//p[.='No tasks created yet']
+
+I can invite Alice to join the workspace
+    Wait Until Page Contains Element  css=div.button-bar.create-buttons a.icon-user-add
+    Click Link  css=div.button-bar.create-buttons a.icon-user-add
+    I can invite Alice to the workspace
+
+I can invite Alice to join the workspace from the menu
+    Wait Until Page Contains Element  link=Functions
+    Click Link  link=Functions
+    Click Link  xpath=//ul[@class='menu']//a[.='Add user']
+    I can invite Alice to the workspace
+
+I can invite Alice to the workspace
+    Wait until page contains  Add user
+    Input Text  css=li.select2-search-field input  alice
+    Click Element  css=span.select2-match
```

Comment:
Any ajaxy select2 robot tests need a 'Wait until element is visible' step before 'Click Element'. This should resolve the issues that jenkins is having.